lib/ace/autocomplete.js

```diff
@@ -143,7 +143,7 @@ var Autocomplete = function() {
         // on IE preventDefault doesn't stop scrollbar from being focussed
         var el = document.activeElement;
         var text = this.editor.textInput.getElement()
-        if (el != text && el.parentNode != this.popup.container
+        if (el != text && this.popup && el.parentNode != this.popup.container
```

Comment:
Well, if the popup is null it has never been shown, so detach shouldn't be necessary  - however having had another look it would be possible for the event handlers to be attached and the popup not be set if an exception should occur somewhere in between, so actually it would be safer to call detach.  I'll update the pull request.